service/tablet_allocator.cc

```diff
@@ -1049,10 +1083,40 @@ class load_balancer {
                     return r2_hosts.contains(dst.host);
                 };
 
-                lblogger.debug("Replica sets of tablets being co-located: ({}: {}), ({}, {})",
-                        global_tablet_id{table, t1.tid}, r1, global_tablet_id{table, t2.tid}, r2);
+                bool rack_awareness = rack_aware_colocation();
```

Comment:
Can evaluate this once, outside `for_each_sibling_tablets` since it applies to all tablets.